templates/includes/sidebar.html

```diff
@@ -1,4 +1,17 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated%}
+    <li>
+      User: <a href="{% url 'taxi:driver-detail' pk=user.id%}">{{user.username}}</a>
+    </li>
+    <li>
+      <a href="{% url 'logout' %}">Logout</a>
+    </li>
+  {% else %}
+    <li>
+     <a href="{% url 'login' %}">Login</a> 
+    </li>
+  {% endif %}
+  <br>
   <li><a href="{% url "taxi:index" %}">Home page</a></li>
   <li><a href="{% url "taxi:manufacturer-list" %}">Manufacturers</a></li>
   <li><a href="{% url "taxi:car-list" %}">Cars</a></li>
```

Comment:
The URL tag uses double quotes for the URL name, which is acceptable but inconsistent with the single quotes used elsewhere. Consider using single quotes for consistency.